### ISSUE TITLE
Split the test fixtures

### ISSUE THREAD

**Issue by cecilemuller** [wildpeaks/package-webpack-configs#657]
There are so many web fixtures nowadays that the file is way too long, and it makes it harder to run only a subset of tests (e.g. only the webworker ones).

Also, Mocha is now able to run in parallel, so I'd like to use it to speed up the web tests (although it most likely require multiple `tmp` folders in that case, meaning multiple `npm install`).

I might later make even more changes if I switch to Rush & pnpm, but that will be its own issue if I do go forward with it.
